SPECS/systemd/systemd-bootstrap.spec

```
@@ -1,13 +1,13 @@
 Summary:        Bootstrap version of systemd. Workaround for systemd circular dependency.
 Name:           systemd-bootstrap
-Version:        249.7
-Release:        3%{?dist}
+Version:        250.3
+Release:        1%{?dist}
 License:        LGPLv2+ AND GPLv2+ AND MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Security
 URL:            https://www.freedesktop.org/wiki/Software/systemd/
-#Source0:       https://github.com/systemd/systemd-stable/archive/v249.7.tar.gz
+#Source0:       https://github.com/systemd/systemd-stable/archive/v250.3.tar.gz
 Source0:        systemd-%{version}.tar.gz
```

Since we're here, could we switch to this "notation":

```suggestion
Source0:        https://github.com/systemd/systemd-stable/archive/v%{version}.tar.gz#/systemd-%{version}.tar.gz
```

It makes automatic parsing easier (extracting the URL and the file name) and `rpmbuild` will still look for a file called `systemd-%{version}.tar.gz`.

---

Good point, I'll make sure to do this on specs I edit going forward. Thanks for the GitHub suggestion!